src/com/mqtt_paho/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ if(FORTE_COM_PAHOMQTT)
       SET(FORTE_COM_PAHOMQTT_LIB "libpaho-mqtt3a.so" CACHE STRING "MQTT Library file in FORTE_COM_PAHOMQTT_LIB_DIR to be linked to forte")
     ELSEIF("${FORTE_ARCHITECTURE}" STREQUAL "Win32")
       SET(FORTE_COM_PAHOMQTT_LIB "paho-mqtt3a.dll" CACHE STRING "MQTT Library file in FORTE_COM_PAHOMQTT_LIB_DIR to be linked to forte")
+      string(FIND ${FORTE_COM_PAHOMQTT_LIB} "static" DETECT)
+      IF(NOT "${DETECT}" STREQUAL "-1")
+        forte_add_link_library("crypt32")
+      ENDIF(NOT "${DETECT}" STREQUAL "-1")
```

Comment:
please dont replicate the if condition in the endif, imho it doen't add to readability, but must be maintained.

also the conditions looks overly complicated
does `NOT DETECT EQUAL -1` not work?

Reply:
I will recheck it.

Reply:
@MartinMelikMerkumians You are correct.